toxcore/group_common.h

```diff
@@ -20,7 +20,15 @@
 #define MAX_GC_NICK_SIZE 128
 #define MAX_GC_TOPIC_SIZE 512
 #define MAX_GC_GROUP_NAME_SIZE 48
-#define MAX_GC_MESSAGE_SIZE 1372
+#define GC_MESSAGE_PSEUDO_ID_SIZE 4
+
+/**
+ * check if in sync with TOX_GROUP_MAX_MESSAGE_LENGTH in tox.h
```

Comment:
This comment is redundant since we check this via static_assert

Reply:
done